src/pineko/check.py

```diff
@@ -43,19 +43,29 @@ def check_grid_and_eko_compatible(pineappl_grid, operators):
         raise ValueError("x grid in pineappl grid and eko operator are NOT compatible!")
 
 
-def check_fonll_b(fns):
+def check_fonll_b(fns, lumi):
     """Checks if the fktable we are computing is a DIS FONLL-B fktable
 
     Parameters
     ----------
         fns : str
             flavor number scheme (from the theory card)
+        lumi : list(list(tuple))
+            luminosity info
 
     Returns
     -------
             : bool
             true if the fktable is a FONLL-B DIS fktable
     """
-    if fns == "FONLL-B":
+    import ipdb
+
+    ipdb.set_trace()
```

Comment:
if possible, avoid to commit debugging instructions (`pre-commit` should actually warns you about, are you using it?)

Reply:
Actually in this case yes, I have used that.

Reply:
Which one of the two?
1. you used and it didn't warn
2. you had a strong need to commit `ipdb`

Because this `pre-commit` action is actually checking the presence of leftover debugging instructions:
https://github.com/N3PDF/pineko/blob/main/.pre-commit-config.yaml#L12

In case of 2., please avoid as much as possible!

Reply:
I used and I didn't get the warning. But ok fine, maybe I am just wrong and I have actually forgot to run pre-commit

Reply:
well, the whole idea of pre-commit is to *not* run it yourself, but to let git do it ...